examples/tina-self-hosted-demo/tina/config.tsx

```diff
@@ -1,35 +1,29 @@
 import { defineStaticConfig } from 'tinacms'
 
-const TINA_TOKEN_KEY = 'tina_token_key'
-
 import { contentBlockSchema } from '../components/blocks/content'
 import { featureBlockSchema } from '../components/blocks/features'
 import { heroBlockSchema } from '../components/blocks/hero'
 import { testimonialBlockSchema } from '../components/blocks/testimonial'
 import { ColorPickerInput } from '../components/fields/color'
 import { iconSchema } from '../components/util/icon'
+import { UsernamePasswordNextAuthProvider } from 'tinacms-next-auth/dist/tinacms'
+
+const isLocal = process.env.TINA_PUBLIC_IS_LOCAL === 'true'
 
 const config = defineStaticConfig({
   contentApiUrlOverride: '/api/gql',
+  clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID!,
   admin: {
-    // auth: {
-    //   // If you wanted to use custom auth
-    //   getToken: async () => {
-    //     return { id_token: 'some-token' }
-    //   },
-    //   logout: async () => {
-    //     localStorage.removeItem(TINA_TOKEN_KEY)
-    //   },
-    //   authenticate: async () => {
-    //     localStorage.setItem(TINA_TOKEN_KEY, 'some-token')
-    //     return true
-    //   },
-    //   getUser: async () => {
-    //     return localStorage.getItem(TINA_TOKEN_KEY)
-    //   },
-    // },
+    authHooks: {}, // TODO why is this required?
+    authCollection: 'user',
```

Comment:
It doesn't _really_ matter to me but it might be nice to handle this the same way we handle isTitle, isBody, ect. 

This is just a suggestion but we could do this instead.

Currently we have

```ts
authCollection: 'user',
```

Would it make more sense to have

```ts
  collections: [
      {
        applicationData: true,
        isAuthCollection: true,
        label: 'Users',
        name: 'user',
        path: 'content/users',
        format: 'md',
        fields: [
          {
            type: 'string',
            label: 'Name',
            name: 'name',
          },
          {
            type: 'string',
            label: 'Email',
            name: 'email',
          },
          {
            type: 'password',
            label: 'Password',
            name: 'password',
          },
        ],
      },
```

We could throw an error if the user has  `isAuthCollection:  true` for more then one collection.

Reply:
Sounds good - I'll update it